Signal/src/call/CallService.swift

```diff
@@ -915,6 +1018,12 @@ protocol CallServiceObserver: class {
         Logger.info("\(self.logTag) in \(#function)")
         SwiftAssertIsOnMainThread(#function)
 
+        guard let callData = self.callData else {
```

Comment:
This is a little unfortunate - if the caller used to have a call, presumably it could instead pass us the CallData instead?

i.e. Can we remove this code and change the parameter to **be** the CallData?